src/scss/includes/colors.scss

```diff
@@ -0,0 +1,49 @@
+/* text */
+$primary_text: #0c0c0e;
+$primary_alternative_text: #ffffff;
+$primary_clear: #c8cbd3;
+$secondary_text : #59595f;
+$secondary_text_clear: #697496;
+$secondary_variant_text : #018475; /* link */
+$grey-black: #0c0c0e;
+$grey-semi-darkness: #59595f;
+
+/* actions */
+$action-blue-base: #1a6aff;
+$action-lighter: #e5eeff;
+
+/* background */
+$background : #ffffff;
+$background_hover : #F4F6FA;
+$background_active: #F4F6FA;
+$background_alternate: #039BE5;
+$surface : #f4f6fa;
+
+/* separator */
+$separator_color : #e0e1e6;
+
+/* error  */
+$error_background : #f8d7da;
+$error_border : #f5c6cb;
+$error_text : #721c24;
+
+:root {
+  --orange-base: #ff9b3c;
```

Comment:
Sorry, I love real CSS vars too, but as long as we support IE11 we can't use them and must keep to Sass vars.

Reply:
What about adding a ponyfill for ie support, like https://github.com/jhildenbiddle/css-vars-ponyfill ?

Reply:
I refer to my previous comment https://github.com/QwantResearch/erdapfel/pull/690#discussion_r446197048: I don't see the need of these vars on the CSS side (and any dependency we don't need is a win).